tests/unit/aroi/test_leaders_scoring.py

```
@@ -316,6 +316,64 @@ def format_bandwidth_with_unit(self, value, unit, decimal_places=1):
+        )
+
+        # Multi-OS, multi-non-EU operator so all four boards have an entry.
+        instance = self._build_relays_instance([
+            ('Linux', 'DE'),      # EU, Linux
+            ('FreeBSD', 'US'),    # non-EU, non-Linux
+            ('FreeBSD', 'US'),    # non-EU, non-Linux (same country)
+            ('OpenBSD', 'JP'),    # non-EU, non-Linux
+        ])
+
+        class _StubFormatter:
+            def determine_unit(self, value):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `determine_unit`

Add return type annotation: `str`

(ANN202)